enterprise/internal/own/ownref.go

```
@@ -373,7 +448,7 @@ func findUserByUsername(ctx context.Context, db edb.EnterpriseDB, handle string)
 func findUserIDByEmail(ctx context.Context, db edb.EnterpriseDB, email string) (int32, error) {
 	// Checking that provided email is verified.
 	verifiedEmails, err := db.UserEmails().GetVerifiedEmails(ctx, email)
-	if err != nil {
+	if err != nil && !errcode.IsNotFound(err) {
```

nit: this type of error is not returned from `GetVerifiedEmails`, we might as well skip this check. An empty slice is returned in a "not found" case

---

Oh! That's interesting. I am wondering if we shouldn't introduce a convention for this. Since sometimes we return not found, and sometimes we don't (nil err and empty result).

---

yep, there is a bit of inconsistency, unfortunately